.gitattributes

```diff
@@ -0,0 +1 @@
+/ui/web/bindata.go binary
```

Comment:
yeah, it's a trick I picked up from grafana plugins where you have compiled files in the codebase that you don't want to have to skim past every time you look at a diff.  I haven't used the `-p` param to `add` or `checkout`, so if it's a problem for that workflow then I'm happy to revert it.